n4js-libs/packages/n4js-cli/README.md

```diff
@@ -17,6 +17,8 @@ For more information about N4JS, please refer to the official [project home](htt
 When installing the package, you can replace the packaged `n4jsc.jar` using the environment variable `N4JSC_JAR` referencing
 either a filesystem path or a HTTP/HTTPS URL.
 
+See [download page](https://download.enfore.com/n4js/).
```

Comment:
We might better link to the Eclipse download page. There we list both the Eclipse and the enfore download pages.

Reply:
The page would be: https://www.eclipse.org/n4js/downloads.html

Reply:
but only the latter links directly to the jars, isn't it?

Reply:
or is there an eclipse jar download page?